platform/features/layout/test/LayoutControllerSpec.js

```diff
@@ -289,6 +308,28 @@ define(
                 expect(controller.getFrameStyle("b"))
                     .not.toEqual(oldStyle);
             });
+
+            it("allows panels to be selected", function() {
+                var childObj = mockCompositionObjects[0];
+
+                controller.select(mockEvent, childObj.getId());
+
+                 // Should have stopped propagation
+                expect(mockEvent.stopPropagation).toHaveBeenCalled();
+
+                // Should have selected the object
+                expect(controller.selected(childObj)).not.toBeFalsy();
```

Comment:
This comment can be removed and code clarified, i.e. expect it `toBe(true)` instead of `not to be falsy`.